content/en/blog/2023/steering-contribution-seat-results/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "2023 Istio Steering Contributor Seats"
+subtitle: Announcing the Istio Steering Committee Contributor Members, 2023.
+description: Announcing the Istio Steering Committee Contributor Members, 2023.
+publishdate: 2023-02-04
+attribution: "Faseela K, for the Istio Steering Committee"
+keywords: [istio,steering,governance,community,contributor]
+---
+
+Based on our [Steering Committee charter](/blog/2020/steering-changes/), Istio shares governance responsibilities between Contribution Seats, selected based on contributions to the project, and Community Seats, elected by the project members. The [Istio Steering Committee](https://github.com/istio/community/blob/master/steering/README.md) consists of 9 proportionally-allocated Contribution Seats, and 4 elected Community Seats.
+
+Last year, we [elected four members](/blog/2022/steering-election-results/) to the community seats, with the committee representing eight different companies. It's now time to reward the companies who fuel our growth with the maximum contributions to Istio, by selecting the [Contributor seat](/blog/2020/steering-changes/#contribution-seats) members to the Istio Steering Committee. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), the Contribution Seat numbers are calculated in February, and the last agreed [metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md) is Contributions, as measured by [devstats](https://istio.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=All).
+
+According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats. As a result we have the following new members joining the Istio Steering Committee, in addition to the existing contributor members.
+
+- [Cale Rath](https://github.com/ctrath) (IBM)
+- [Justin Pettit](https://github.com/justinpettit) (Google)
+- [John Howard](https://github.com/howardjohn) (Google)
+- [Zhonghu Xu](https://github.com/hzxuzhonghu) (Huawei)
+
+We wish to offer our thanks to Srihari Angaluri, Kebe Liu and Louis Ryan, all long-time contributors to the Istio project, whose term will be ending now.
```

Comment:
All three leaving for different reasons.  Does saying "long-time contributors" add anything? 

```suggestion
Our sincerest thanks to Srihari Angaluri, Kebe Liu and Louis Ryan, all long-time contributors to the Istio project, whose terms have come to an end.
```